README.md

```diff
@@ -1,164 +1,123 @@
 ---
 author: Polkadot developers
-revision: 0.3.1
+revision: 0.4.0
```

Comment:
I would remove this

Or is this documentation revision version is used anywhere? We have versioning of crates. Crate README is pushed to autogenerated docs also afaik, it's weird that there will be different versions in the final html. And FWIW revisions are usually numbered with integers.

Reply:
Yeah thought it was weird. Will just remove it.